### ISSUE TITLE
Update API endpoints

### ISSUE THREAD

**Issue by ozdemirburak** [ozdemirburak/skyscanner-php#13]
This package is now outdated since Skyscanner now uses v3 API endpoints. 

Example call:

> POST https://partners.api.skyscanner.net/apiservices/v3/flights/live/search/create

Each endpoint needs to be updated and tested.

For the API key, you can check the testing key:

https://developers.skyscanner.net/docs/getting-started/authentication

**Comment by KirDE** [comment]
This will require alsomst comlete rework of the library and different input and output data format, which will lead to incompatibility with existing systems. But anyway support for v1 API will end in Jan 2023, so this change is needed.

**Comment by ozdemirburak** [comment]
That is correct. Thanks for your feedback @KirDE. Are you interested in this issue?
